src/mono/mono/mini/mini-llvm.c

```diff
@@ -8281,6 +8281,16 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 			break;
 		}
 
+		case OP_SSE2_PSHUFD_IMM: {
+			LLVMValueRef mask = create_const_vector_4_i32 (
```

Comment:
This is not not actually used.

Reply:
Also, what is the problem with the non-imm version. LLVM should be able to optimize it down to the same code. In general, detecting immediate arguments is hard because the JIT might insert copies etc.